packages/ckeditor5-dev-build-tools/src/plugins/replace.ts

```diff
@@ -22,26 +24,56 @@ export interface RollupReplaceOptions {
 	replace: Array<[ RegExp | string, string ]>;
 }
 
-export function replace( pluginOptions: RollupReplaceOptions ): Plugin {
+export function replaceImports( pluginOptions: RollupReplaceOptions ): Plugin {
 	const options: Required<RollupReplaceOptions> = Object.assign(
 		{ replace: [] },
 		pluginOptions
 	);
 
+	function isModule( node: Node ): node is ImportDeclaration | ExportAllDeclaration | ExportNamedDeclaration {
+		return [
+			'ImportDeclaration',
+			'ExportAllDeclaration',
+			'ExportNamedDeclaration'
+		].includes( node.type );
+	}
+
 	return {
-		name: 'cke5-replace',
+		name: 'cke5-import-replace',
```

Comment:
```suggestion
		name: 'cke5-replace-import',
```

To be consistent with the function name `replaceImports`.